_posts/2020-05-13-aria-has-perfect-support.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "Myth: ARIA Has Perfect Support"
+description: "ARIA is a very useful tool for helping to enhance accessible web experiences, but it should only be used when no other options are available."
+author: eric_bailey
+published: true
+categories:
+  - Myths
+further_reading:
+  - url: https://a11ysupport.io/
+    title: "Accessibility Support"
+    source: a11ysupport.io
+  - url: https://www.powermapper.com/tests/screen-readers/aria/
+    title: "WAI-ARIA Screen reader compatibility"
+    source: PowerMapper Software
+  - url: https://developer.paciellogroup.com/blog/2018/07/short-note-on-aria-support/
+    title: "Short note on ARIA support"
+    source: The Paciello Group
+  - url: https://css-tricks.com/aria-spackle-not-rebar/
+    title: "ARIA is Spackle, Not Rebar "
+    source: CSS-Tricks
+---
+
+[Accessible Rich Internet Applications (ARIA)](/posts/getting-started-aria/) is an extension of HTML. It is a technology that adds roles, states, and properties that are designed to help with the accessibility of websites and web apps.
+
+
+## Background
+
+ARIA works by supplementing, adding, removing, or overriding information the browser uses to communicate with assistive technology. For example, adding `aria-hidden="true"` to a HTML element will tell the browser not to report the element, or any of its children to assistive technology. 
+
+`aria-hidden="true"` can be useful for situations where you want to remove something that might normally get reported, such as a SVG icon inside a button. If the button already has a text label, removing the icon helps make sure that the experience is easy to understand:
+
+``` html
+<button type="button">
+  <svg aria-hidden="true" focusable="false">
+    <!-- SVG code -->
+  </svg>
+  Save
+</button>
+```
+
+ARIA can also be used for interactive controls such as accordions, tabs, and modals. A thoughtful, skilled application of ARIA can help communicate things like if an accordion is in an expanded state, which tab is currently being viewed, or what is and is not modal content.
+
+Nearly all ARIA is inserted and updated dynamically via JavaScript. Very little of it is “hard coded” into the HTML it is manipulating. There are many ways to write this JavaScript. You can use “vanilla” JavaScript, or specialized frameworks such as [Ember](https://emberjs.com/), [Angular](https://angular.io/), [React](https://reactjs.org/), [Vue](https://vuejs.org/), [Svelte](https://svelte.dev/), etc. ARIA is compatible with all these frameworks because [it is a standard](https://w3c.github.io/aria/) they observe.
```

Comment:
"Nearly all ARIA is inserted and updated dynamically via JavaScript."

should "is inserted" be "should be inserted"???

Reply:
Ideally 😅